ambry-account/src/test/java/com/github/ambry/account/MockNotifier.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2017 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package com.github.ambry.account;
+
+import com.github.ambry.commons.Notifier;
+import com.github.ambry.commons.TopicListener;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+
+
+/**
+ * A mock implementation of {@link Notifier}.
+ */
+public class MockNotifier implements Notifier<String> {
```

Comment:
since this is not doing any `String` specific stuff, would be good to make this also a generic template class. What do you think?